src/components/Releases.js

```diff
@@ -0,0 +1,144 @@
+/********************************************************************************
+ * Copyright (C) 2021 EclipseSource and others.
```

Comment:
minor: should it not be 2022?

```suggestion
 * Copyright (C) 2022 EclipseSource and others.
```